test/mocks/network/mocks.h

```diff
@@ -428,7 +428,9 @@ class MockTransportSocket : public TransportSocket {
   MockTransportSocket();
   ~MockTransportSocket();
 
-  MOCK_METHOD1(setTransportSocketCallbacks, void(TransportSocketCallbacks& callbacks));
+  void setTransportSocketCallbacks(TransportSocketCallbacks& callbacks) override {
```

Comment:
Keep this (and `callbacks`) below as a MOCK and do ON_CALL(...).WillByDefault in constructor like:
https://github.com/envoyproxy/envoy/pull/4153/files#diff-f1d609046babf8223367556fbdda1721R206

This will allow test to do EXPECT_CALL etc.

Reply:
done